drillsrs/cmd/import_.py

```diff
@@ -1,14 +1,36 @@
 import argparse
 import json
 import sys
+import io
 from typing import IO, Any, Optional
 
 from dateutil.parser import parse as parse_date
+from anki_export import ApkgReader
+from lxml import etree
 
 from drillsrs import db, scheduler, util
 from drillsrs.cmd.command_base import CommandBase
 
 
+def tojson(filepath):
+    with ApkgReader(filepath) as apkg:
+        temp = apkg.export()
+    temp = temp[list(temp)[0]]
+    x = {"name":temp[1][3], "description":None, "tags":[], "cards":[]}
+    counter = 1
+    for s in temp[1:]:
+        card = {"active":False,"activation_date":None,"tags":[],"user_answers":[]}
+        card["id"] = counter
+        q = etree.HTML(s[0])
+        card["question"] = etree.tostring(q, encoding='unicode', method='text')
+        a = etree.HTML(s[1])
+        card["answers"] = [" "] if a is None else [etree.tostring(a, encoding='unicode', method='text')]
```

Comment:
Why a space?

Reply:
Some time when removeing tranformaing answer from html to plain text it return none and lxml etree give me 
`TypeError: Type 'NoneType' cannot be serialized.` 
so i left as empty answer for now
I need to find better solution